test-clients/src/main/java/com/hedera/services/bdd/suites/regression/SteadyStateThrottlingCheck.java

```diff
@@ -106,22 +113,25 @@ public static void main(String... args) {
 	@Override
 	protected List<HapiApiSpec> getSpecsInSuite() {
 		return List.of(
-				new HapiApiSpec[] {
-						setArtificialLimits(),
+				new HapiApiSpec[]{
+						setArtificialLimits(artificialLimits),
 						checkTps("Xfers", expectedXferTps, xferOps()),
 						checkTps("FungibleMints", expectedFungibleMintTps, fungibleMintOps()),
 						checkTps("ContractCalls", expectedContractCallTps, scCallOps()),
 						checkTps("CryptoCreates", expectedCryptoCreateTps, cryptoCreateOps()),
-//						checkTps("NftMints", expectedNftMintTps, nftMintOps()),
+						checkTps("NftMints", expectedNftMintTps, nftMintOps()),
 						checkBalanceQps(1000, expectedGetBalanceQps),
+						setArtificialLimits(artificialGasLimits),
+						checkGasThrottledTps("ContractCalls", expectedContractCallSmallGasTps, scCallSmallTxsBlockedFromFE()),
+						checkGasThrottledTps("ContractCalls", expectedContractCallLargeGasTps, scCallLargeTxsBlockedFromFE()),
+						checkGasThrottledTps("ContractCalls", expectedContractCallSmallGasTps, scCallSmallTxsBlockedFromBE()),
+						checkGasThrottledTps("ContractCalls", expectedContractCallLargeGasTps, scCallLargeTxsBlockedFromBE()),
 						restoreDevLimits(),
 				}
 		);
 	}
 
-	private HapiApiSpec setArtificialLimits() {
-		var artificialLimits = protoDefsFromResource("testSystemFiles/artificial-limits.json");
-
+	private HapiApiSpec setArtificialLimits(ThrottleDefinitions artificialLimits) {
 		return defaultHapiSpec("SetArtificialLimits")
 				.given().when().then(
 						fileUpdate(THROTTLE_DEFS)
```

Comment:
I think we need to add 								
.hasKnownStatusFrom(SUCCESS, SUCCESS_BUT_MISSING_EXPECTED_OPERATION)
to the fileUpdate Op since this is returned sometimes from Hedera